graph/iterator/linksto_iterator.go

```diff
@@ -176,19 +187,34 @@ func (it *LinksTo) Next() bool {
 	return it.Next()
 }
 
+func (it *LinksTo) Err() error {
+	return it.err
+}
+
 func (it *LinksTo) Result() graph.Value {
 	return it.result
 }
 
-// Close our subiterators.
-func (it *LinksTo) Close() {
-	it.nextIt.Close()
-	it.primaryIt.Close()
+// Close our subiterators.  It closes all subiterators it can, but
```

Comment:
Do you mind clarifying?  I'm not seeing anything wrong here, but I might be missing something.